README.md

```diff
@@ -89,24 +92,32 @@ Which of the following represents a correct workflow for submitting a PR on a no
 
 Select 1:
 ```
+I think it's the third one, since if you're working in a non-master branch, a fork isn't needed.
+
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
 [] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
-[] git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
+[x] git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
 [] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git pull; create pull request
 ```
```

Comment:
The correct answer is the first one. Because you're submitting an assignment and you can only read from the GA repo, you'll need to fork it first so that you can later push your changes.